calico-cloud_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx

```diff
@@ -0,0 +1,35 @@
+xx July 2023
+
+### New and improved
+
+#### New policy recommendations engine for namespace isolation
+{{prodname}} has added a new policy recommendations engine that automatically generates staged policies for namespace isolation within your cluster. [Policy recommendations](network-policy/recommendations/policy-recommendations.mdx).
+
+#### Destination-based routing for egress gateways
+{{prodname}} introduces a new mode for egress gateways that can leverage destination-based routing. Destination-based routing for egress gateways allows operators associated with a destination that is external to a Kubernetes cluster (for example, IP address or CIDR), to a specific egress gateway deployment. [Egress gateways](networking/egress/egress-gateway-on-prem.mdx).
+
+#### Support for DNS rules in clusters using NodeLocal DNSCache
+{{prodname}} has added support for DNS rules in clusters using NodeLocal DNSCache. Also related, there is new documentation on using Calico policy to secure DNS traffic within the cluster with NodeLocal DNSCache enabled. [Use NodeLocal DNSCache in your cluster](networking/configuring/node-local-dns-cache.mdx).
+
+#### Improved UI for configuring Workload-based Web Application Firewall (WAF)
+{{prodname}} includes updates to the UI that allows you to select which services are enabled for the Workload-based Web Application Firewall. [Web application firewall](threat/web-application-firewall.mdx).
+
+#### Wireguard support for AKS and EKS with Calico CNI
+{{prodname}} now offers official support for Wireguard when using Microsoft AKS or Amazon EKS with Calico CNI. This mode of deployment offers performance benefits and a more efficient routing table compared to using cloud provider CNIs.  [Encrypt data in transit](compliance/encrypt-cluster-pod-traffic.mdx).
+
```

Comment:
Add release notes on RBAC updates:
Additional custom roles for Calico Cloud users
Users can now create custom role-based access controls for two new roles, "Usage Metrics" and "Image Assurance Admin"

Reply:
Added.